tests/test_filtering.py

```diff
@@ -62,6 +64,13 @@ class WidgetListView(GenericModelView):
 
         filtering = Filtering(
             color=operator.eq,
```

Comment:
then everything could be simplified using kwargs, instead of checking if the first param is a string or a callable